controllers/keystoneapi_controller.go

```diff
@@ -1205,11 +1227,49 @@ func (r *KeystoneAPIReconciler) generateServiceConfigMaps(
 		endptConfig := map[string]interface{}{}
 		endptConfig["ServerName"] = fmt.Sprintf("%s-%s.%s.svc", instance.Name, endpt.String(), instance.Namespace)
 		endptConfig["TLS"] = false // default TLS to false, and set it bellow to true if enabled
+		endptConfig["EnableFederation"] = enableFederation
 		if instance.Spec.TLS.API.Enabled(endpt) {
 			endptConfig["TLS"] = true
 			endptConfig["SSLCertificateFile"] = fmt.Sprintf("/etc/pki/tls/certs/%s.crt", endpt.String())
 			endptConfig["SSLCertificateKeyFile"] = fmt.Sprintf("/etc/pki/tls/private/%s.key", endpt.String())
 		}
+
+		if enableFederation {
```

Comment:
I think we could just add all this to the templateParameters as they are not endpoint specific like the certs. like the httpd ProcessNumber

Reply:
or at least query the ospSecret out side the for loop